src/converters/lintConfigs/rules/ruleConverters/eslint-plugin-react/jsx-space-before-trailing-slash.ts

```diff
@@ -0,0 +1,17 @@
+import { RuleConverter } from "../../ruleConverter";
+
+export const convertJsxSpaceBeforeTrailingSlash: RuleConverter = () => {
+    return {
+        rules: [
+            {
+                ruleArguments: [
+                    {
+                        beforeSelfClosing: "always",
```

Comment:
I believe this is set by default by this rule 🤔 https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-tag-spacing.md#rule-details

Reply:
Oh yes thanks. And I think the rest of the config options will need to be set to `"allow"` to avoid breaking changes...